datastore/tasks/tasks_test.go

```diff
@@ -101,6 +102,18 @@ func TestList(t *testing.T) {
 	}
 }
 
+func TestCreateClientWithDatabase(t *testing.T) {
+	test := testutil.SystemTest(t)
+	projectID := test.ProjectID
+	databaseName := "customdb"
```

Comment:
do we need to ensure this database exists already, or will the client create it automatically?

Reply:
The client does not create database automatically

Reply:
Good to know. Let me get that database set up....